DOCUMENTATION.md

```diff
@@ -2188,7 +1862,7 @@ hz.getPNCounter('myPNCounter').then(function (counter) {
 });
 ```
 
-## 7.4.13. Using Flake ID Generator
+### 7.4.10. Using Flake ID Generator
 
 Hazelcast `FlakeIdGenerator` is used to generate cluster-wide unique identifiers. Generated identifiers are long primitive values and are k-ordered (roughly ordered). IDs are in the range from 0 to `2^63-1` (maximum signed long value). For details, see the [FlakeIdGenerator section](https://docs.hazelcast.org/docs/latest/manual/html-single/index.html#flakeidgenerator) in the Hazelcast IMDG Reference Manual.
```

Comment:
nit: change 0 to `0`. Also, "maximum signed long value" could be changed to "maximum value for Java's `long` type.